modules/packages/UnitTest.chpl

```diff
@@ -19,19 +19,24 @@
  */
 
 /*
-Module UnitTest provides support for automated testing in Chapel.
-Any function of the form
+The UnitTest module provides support for automated testing in Chapel.
+
+The UnitTest module is intended to be used in with the `mason test
+<https://chapel-lang.org/docs/tools/mason/mason.html>`_ command, which
```

Comment:
is the trailing _ a part of chpldoc syntax?

Reply:
rst syntax specifically, but yes.